test/executor.test.js

```diff
@@ -906,6 +906,62 @@ describe('executor', function() {
     });
   });
 
+  describe('when booting with lazy connect', function() {
+    var sampleInstruction = someInstructions({
```

Comment:
Please rename to indicate that this should be treated as an immutable constant, e.g. `SAMPLE_INSTRUCTIONS`.